paasta_tools/tron_tools.py

```diff
@@ -98,11 +99,16 @@
 DEFAULT_AWS_REGION = "us-west-2"
 EXECUTOR_TYPE_TO_NAMESPACE = {
     "paasta": "tron",
-    "spark": "paasta-spark",
+    "spark": "tron",
 }
 DEFAULT_TZ = "US/Pacific"
 clusterman_metrics, _ = get_clusterman_metrics()
-EXECUTOR_TYPES = ["paasta", "ssh"]
+EXECUTOR_TYPES = ["paasta", "ssh", "spark"]
+SPARK_AWS_CREDS_PROVIDER = "com.amazonaws.auth.WebIdentityTokenCredentialsProvider"
+SPARK_EXECUTOR_NAMESPACE = "paasta-spark"
+SPARK_DRIVER_POOL = "stable"
+SPARK_JOB_USER = "TRON"
```

Comment:
```suggestion
SPARK_JOB_USER = "TRON"
PROMETHEUS_SHARD="ml-compute"
```

Reply:
Same as the above comment => `SPARK_PROMETHEUS_SHARD`